packages/service-clients/tinylicious-client/package.json

```diff
@@ -79,7 +79,7 @@
 		"test": "npm run test:realsvc",
 		"test:realsvc": "npm run test:realsvc:tinylicious",
 		"test:realsvc:tinylicious": "start-server-and-test start:tinylicious:test 7070 test:realsvc:tinylicious:run",
-		"test:realsvc:tinylicious:run": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit",
+		"test:realsvc:tinylicious:run": "mocha --exit",
```

Comment:
This package doesn't use mocha-test-setup, we should add it if we want to make this change, right?

Reply:
Good catch. As is, CI is passing, but this should be erroring when it runs. Additionally, no matter what I do (even revert it to how it was on main) the two "can process data corruption events" are timing out.

If I pick a recent passing pipeline run and search it for these tests, they don't appear in the summary or the log so somehow this packages tests don't seem to be running at all on CI?

Reply:
I have reverted this change. These tests don't run on CI either way, and also fail either way, so I'll leave it to be fixed by #25344 and it can be updated to follow the new conventions either as part of that or later.